elements/lisk-transactions/src/create_sendable.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { validateAddress, validatePublicKey } from '@liskhq/lisk-validator';
+
+import { TransactionJSON } from './transaction_types';
+
+import {
+	getAddressFromPublicKey,
+	getPrivateAndPublicKeyBytesFromPassphrase,
+} from '@liskhq/lisk-cryptography';
+
+const validateRequiredInputs = (
+	type: number,
+	passphrase: string,
+	receipientId?: string | null,
+	recipientPublicKey?: string,
+): void => {
+	if (!(receipientId || recipientPublicKey)) {
+		throw new Error(
+			'Either recipientId or recipientPublicKey must be provided.',
+		);
+	}
+
+	if (typeof receipientId !== 'undefined' && receipientId !== null) {
+		validateAddress(receipientId);
+	}
+
+	if (typeof recipientPublicKey !== 'undefined') {
+		validatePublicKey(recipientPublicKey);
+	}
+
+	if (
+		receipientId &&
+		recipientPublicKey &&
+		receipientId !== getAddressFromPublicKey(recipientPublicKey)
+	) {
+		throw new Error('recipientId does not match recipientPublicKey.');
+	}
+
+	if (!passphrase) {
+		throw new Error(
+			'Cannot sign a transaction without a passphrase. Specify your passphrase as in the input object (and optional second passphrase)',
+		);
+	}
+
+	if (!type || typeof type !== 'number') {
+		throw new Error('type must be provided.');
+	}
+};
+
+const asJSON = (transaction: Partial<TransactionJSON>): string =>
```

Comment:
prefer `toString()` or `stringify`

Reply:
Replaced with the direct JSON.stringify call